src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3513,6 +3513,13 @@ protected override void VisitStatement(BoundStatement statement)
             return null;
         }
 
+        public override BoundNode? VisitCollectionExpressionSpreadElement(BoundCollectionExpressionSpreadElement node)
+        {
+            base.VisitCollectionExpressionSpreadElement(node);
+            SetResultType(node, default);
```

Comment:
Why would this be default? Can it ever actually be `null`?

Reply:
> Can it ever actually be `null`?

Are you asking if `node.Type` can be `null`? With this PR, the spread element type is always `null` since the `BoundExpression.Type` is not used.

It looks like we use the same `SetResultType(node, default)` for `BoundThrowExpression`, which seems like a similar case where the `BoundExpression.Type` is not meaningful.